test/e2e/case5_policy_automation_test.go

```diff
@@ -137,9 +142,33 @@ var _ = Describe("Test policy automation", func() {
 				Expect(err).To(BeNil())
 				_, err = utils.KubectlWithOutput("get", "ansiblejobs", "-n", testNamespace)
 				Expect(err).Should(BeNil())
+				spec := ansiblejobList.Items[0].Object["spec"]
+				extraVars := spec.(map[string]interface{})["extra_vars"].(map[string]interface{})
+
+				return len(extraVars)
+			}, 30, 1).Should(Equal(8))
+
+			By("Check each violation context field in extraVars")
+			ansiblejobList, err := clientHubDynamic.Resource(gvrAnsibleJob).Namespace(testNamespace).List(
+				context.TODO(), metav1.ListOptions{},
+			)
+			Expect(err).To(BeNil())
+			spec := ansiblejobList.Items[0].Object["spec"]
+			extraVars := spec.(map[string]interface{})["extra_vars"].(map[string]interface{})
+			Expect(extraVars["policy_name"]).To(Equal("case5-test-policy"))
+			Expect(extraVars["namespace"]).To(Equal("policy-propagator-test"))
```

Comment:
I think `policy-propagator-test` is stored as a global variable. Could you please use the variable if this is the case?